modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/uiselect/plugin.js

```diff
@@ -24,12 +24,24 @@
 
 			if (data.icon) {
 				output =
-					'<span class="cke_button"><span class="cke_button_icon cke_button__{icon}_icon"' +
+					'<a class="cke_button"' +
+					'tabindex="-1"' +
+					'title="{title}"' +
+					'hidefocus="true"' +
+					'role="button"' +
+					'"><span class="cke_button_icon cke_button__{icon}_icon"' +
 					' >&nbsp;</span>';
```

Comment:
Can you use a template literal string here instead of concatenation?

Reply:
We need use ES5 here. So, We can't have template literals :/

Reply:
Just curious and maybe I should already know this, but what makes this file have to be ES5?

Reply:
That's a good question. I think because We want to backport those things and the modularization of ckeditor4(see `requires: [` at the bottom of the file). Maybe @julien or @markocikos know more about it